src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const threeDigitSeparator = (number) => {
+  const stringNumber = number.toString();
+  const result = [];
+
+  for (let i = 0; i < stringNumber.length; i++) {
+    if (i % 3 === 0) {
+      result.push(',');
+    }
+    result.push(stringNumber.split('').reverse()[i]);
```

Comment:
Maybe it's better to split and reverse stringNumber before cycle, instead of doing it in every iteration

Reply:
yes, thank you! i don't know how i missed that one.